src/components/Footer/Footer.module.css

```diff
@@ -0,0 +1,15 @@
+.footer {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.unfinishedDeal {
+  font-size: 21px;
+  color: #f50057;
+}
+
+.filterButton {
+  margin-bottom: 10px;
+}
```

Comment:
Codacy found an issue: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/tr-todo/pullRequest?prid=5765946)